site/pages/home/homepage-top-slice/style.css

```diff
@@ -0,0 +1,58 @@
+@value white from "../../../css/_colors.css";
+@value mediumScreen, largeScreen from "../../../css/_breakpoints.css";
+
+.homepageTopSlice {
+  padding: 50px 60px 100px 60px;
+  text-align: center;
+  background-color: white;
+}
+
+.badgerSlogan {
+  composes: fontXL from "../../../css/typography/_fonts.css";
+  font-weight: bold;
```

Comment:
is font-weight bold not a feature of fontXL on purpose? same for fontM on line 17 - can you check with the designers about the reusability of these styles?